src/components/Loader/Loader.example.js

```diff
@@ -5,7 +5,8 @@
  * This source code is licensed under the license found in the LICENSE file in
  * the root directory of this source tree.
  */
-import { center } from 'stylesheets/base.scss';
```

Comment:
Can you revert these imports throughout the project back into 1 line?

Reply:
No, I believe it's no longer supported in css-loader (or sass-loader?).

Reply:
Oh, alright then.